### ISSUE TITLE
consider re-exporting Rng trait

### ISSUE THREAD

**Issue by vitiral** [AltSysrq/proptest#17]
This is a pretty minor papercut.

As it is an extra dependency (`rand`) is required to make functions which can use the rng passed by `Strategy::prop_perturb`

It would be nice to not require this dependency. This also protects against (rare) cases where someone depends on multiple versions of `rand`. They could specify `proptest::Rng` specifically if they needed to.

**Comment by AltSysrq** [comment]
The problem had occurred to me when I wrote that code, but I didn't think there was such a simple solution.  Would you happen to have an example of another crate that takes this approach? I couldn't find anything on the topic after some brief searching, but I'd like to adhere to convention if any exists here. (If not, that's fine, I'll figure something out.)

In any case, thanks for the suggestion!

**Comment by vitiral** [comment]
I've seen it before, but I'm not sure how common it is.
- [itertools](https://docs.rs/itertools/0.7.4/itertools/) rexports the `Either` type.
- [jsonrpc-core](https://paritytech.github.io/jsonrpc/jsonrpc_core/index.html) is where I first saw it, in that it rexports [`serde_json::Value`](https://paritytech.github.io/jsonrpc/jsonrpc_core/types/enum.Value.html). This proved to be very important when serde was moving to 1.0 since jsonrpc-core was slower than some other libraries and I had to convert between the two different serde types to use the library. I don't know how I would have done it had jsonrpc_core not rexported  `Value`,`to_string` and `to_value`.

A mark against is that neither `serde_json` or `toml` rexport the `Serialize` or `Deserialize` types.
